modules/dashboards/library/Dashboards/Form/DashletForm.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Icinga\Module\Dashboards\Form;
+
+use Icinga\Module\Dashboards\Model\Database;
+use ipl\Sql\Select;
+use ipl\Web\Compat\CompatForm;
+
+class DashletForm extends CompatForm
+{
+    use Database;
+
+    public function fetchDashboards()
+    {
+        $dashboards = [];
+
+        $select = (new Select())
+            ->columns('*')
+            ->from('dashboard');
+
+        $data = $this->getDb()->select($select);
+
+        foreach ($data as $dashboard) {
+            $dashboards[$dashboard->id] = $dashboard->name;
+        }
+
+        return $dashboards;
+    }
+
+    public function fetchNewDashboardId()
```

Comment:
Let's make this a more utility like function with the following definition `createDashboard(string name): int id`